src/Ojs/CoreBundle/Service/Twig/OjsExtension.php

```diff
@@ -539,10 +539,18 @@ public function getAuthorsInfo(Author $author)
 
         $text = '
         <p id="author$' . $author->getId() . '">
-        <b>' . $this->translator->trans('author') . ': </b>' . $fullName . '</br>
-        <b>' . $this->translator->trans('email') . ': </b>' . $email . '</br>
-        <b>' . $this->translator->trans('institution') . ': </b>' . $institution . '</br>
-        <b>' . $this->translator->trans('country') . ': </b>' . $author->getCountry() . '</p><hr>';
+        <b>' . $this->translator->trans('author') . ': </b>' . $fullName . '</br>';
+
+        if (!empty($this->translator->trans('email'))){
```

Comment:
probably you want to check if empty `$email` variable. why did you checked translation existance? @hmert